app/models/grda_warehouse/hud/client.rb

```diff
@@ -355,11 +355,7 @@ class Client < Base
       when :release_present
         where(housing_release_status: [full_release_string, partial_release_string])
       when :active_clients
-        range = GrdaWarehouse::Config.cas_sync_range
-        # Homeless or Coordinated Entry
-        enrollment_scope = GrdaWarehouse::ServiceHistoryEnrollment.in_project_type([0, 1, 2, 4, 8, 14]).
-          with_service_between(start_date: range.first, end_date: range.last)
-        where(id: enrollment_scope.select(:client_id))
+        where(id: GrdaWarehouse::ServiceHistoryEnrollment.cas_active_clients.select(:client_id))
```

Comment:
this wasn't using `ineligible_uses_extrapolated_days` -- not sure if I am fixing a bug or breaking something